README.md

```diff
@@ -3,6 +3,7 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/ivadomed/ivadomed/badge.svg?branch=master)](https://coveralls.io/github/ivadomed/ivadomed?branch=master)
 ![](https://github.com/neuropoly/ivadomed/workflows/Python%20package/badge.svg)
+[![Documentation Status](https://readthedocs.org/projects/ivado-medical-imaging/badge/?version=latest)](https://ivadomed.org/en/latest/?badge=latest)
```

Comment:
why "?badge=latest" in the URL?

Reply:
sorry just took the default setting for the badge. Do you want to use stable ?

Reply:
i think so, yes.

Reply:
fixed in 6641660